documentation/sphinx/source/configuration.rst

```diff
@@ -498,6 +498,20 @@ To change the storage engine, use the ``configure`` command of ``fdbcli``. For e
 	
     When using the ``memory`` engine, especially with a larger memory limit, it can take some time (seconds to minutes) for a storage machine to start up. This is because it needs to reconstruct its in-memory data structure from the logs stored on disk.
 
+.. _configuration-storage-engine-redwood:
```

Comment:
Thanks!

Can you update line 458 and 460 where only 2 engines are mentioned?

Reply:
my bad, missed them. Done.